i18n/en.js

```diff
@@ -936,14 +944,14 @@ const en = {
             "support": "Call 1-877-505-0253 (6:00 a.m-8:00 p.m. PT on weekdays and 7:00 a.m.-4:00 p.m. PT on weekends)"
         },
         "bswh": {
-            "donatingSamples": "As part of Connect, we ask you to donate blood, urine, and mouthwash samples and complete a short survey.",
+            "donatingSamples": "As part of Connect, we ask you to donate blood, urine, and mouthwash samples and complete a short surveys.",
```

Comment:
Is `surveys` supposed to be singular or plural?

Reply:
fixed